### ISSUE TITLE
Unable to find IAnimationManager for 'Microsoft.Maui.Controls.Shapes.Path' exception for page with Uranium Checkbox

### ISSUE THREAD

**Issue by JorisZwaenepoel** [enisn/UraniumUI#705]
Today, I suddenly get this error:

Unable to find IAnimationManager for 'Microsoft.Maui.Controls.Shapes.Path'

when loading a page that has a uranium UI checkbox on it.
If I remove the checkbox from the page, the page loads correctly.

This started out of nowhere, and the app stull runs correctly on 3 other computers.
I have all my tools updated to the latest version (VS 2022, version 17.10.3) and Uranium UI version 2.8.1.

I tried getting the source code for the solution fresh from git in a new folder, but the problem is stille here.
Any ideas what could be causing this?

![image](https://github.com/enisn/UraniumUI/assets/16609982/ba55940a-ace6-47e7-90c6-7d27ee48f2e3)

**Comment by enisn** [comment]
Seems similar to https://github.com/enisn/Xamarin.Forms.InputKit/issues/359

Can you share usage of CheckBox in your page? An initial value might cause it

**Comment by enisn** [comment]
Ok found it, it should be fixed by PR https://github.com/enisn/Xamarin.Forms.InputKit/pull/369

**Comment by JorisZwaenepoel** [comment]
I suppose this fix will be included in the next release?
Can I ask about the release date for the next version, of would you recommend I clone the latest version from github and use that instead of the NuGet package?

**Comment by enisn** [comment]
It's released in InputKit 4.4.4 version. You can update only [InputKit.Maui](https://www.nuget.org/packages/InputKit.Maui) package to 4.4.4 in your project.

Normally you don't have a reference to inputkit, it's referenced by UraniumUI but instead of waiting UraniumUI release, you can simply add InputKit.Maui reference to your app

```xml
<PackageReference Include="InputKit.Maui" Version="4.4.4" />
```
